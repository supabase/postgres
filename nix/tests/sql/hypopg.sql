create schema v;

create table v.samp(
  id int
);

select 1 from hypopg_create_index($$
  create index on v.samp(id)
$$);

drop schema v cascade;


