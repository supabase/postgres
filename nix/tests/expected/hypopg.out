create schema v;
create table v.samp(
  id int
);
select 1 from hypopg_create_index($$
  create index on v.samp(id)
$$);
 ?column? 
----------
        1
(1 row)

drop schema v cascade;
NOTICE:  drop cascades to table v.samp
